lib/Migration/Version000920Date20250422000000.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/** @noinspection PhpUnused */
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\Tables\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version000920Date20250422000000 extends SimpleMigrationStep {
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('tables_columns')) {
+			$table = $schema->getTable('tables_columns');
+
+			if (!$table->hasColumn('text_unique')) {
```

Comment:
Please add the column also to the first migration that introduces `tables_columns`

Reply:
done